botoffliner.lua

```diff
@@ -336,20 +336,20 @@ function ExecuteCommand( tUser, sCmd, sData )
 			Core.SendPmToUser( tUser, tCfg.sBotName, "Sorry! The user "..tBreak[1].." is not a moderator." )
 			return true
 		end
-		tOffliner.delmod( tUser, tBreak[1] )
-		sAllModerators, sAllCategories = tFunction.Connect()
-		local sChatMessage = "Moderator removed: "..tBreak[1].." ."
-		tFunction.SendToAll( tUser.sNick, sChatMessage )
-		SendToRoom( tUser.sNick, sChatMessage, tCfg.sReportBot )
-		if not RegMan.GetReg( tBreak[1] ) then
-			Core.SendPmToUser( tUser, tCfg.sBotName, "Sorry! The user "..tBreak[1].." is unregistered at the moment." )
-			return true
-		elseif not tProfiles.AllowAdmin[tUser.iProfile] then
-			RegMan.ChangeReg( tBreak[1], RegMan.GetReg(tBreak[1]).sPassword, tCfg.iRegProfile )
-			return true
+		for iIndex,sNic in ipairs( tBreak ) do
+		        tOffliner.delmod( tUser, tBreak[iIndex] )
+			sAllModerators, sAllCategories = tFunction.Connect()
+			local sChatMessage = "Moderator removed: "..tBreak[iIndex].." ."
```

Comment:
Instead of displaying a message every time a moderator is removed, see if you can display a single message with the names of moderators removed. Something like:
```Moderator(s) removed: Nick1 Nick2 Nick3 ...`